packages/expect/src/__tests__/toThrowMatchers.test.ts

```diff
@@ -278,6 +278,60 @@ describe.each(['toThrowError', 'toThrow'] as const)('%s', toThrow => {
     });
   });
 
+  describe('error message and cause', () => {
+    const errorA = new Error('A');
+    const errorB = new Error('B', {cause: errorA});
+    const expected = new Error('good', {cause: errorB});
+
+    describe('pass', () => {
+      test('isNot false', () => {
+        jestExpect(() => {
+          throw new Error('good', {cause: errorB});
+        })[toThrow](expected);
+      });
+
+      test('isNot true, incorrect message', () => {
+        jestExpect(() => {
+          throw new Error('bad', {cause: errorB});
+        }).not[toThrow](expected);
+      });
+
+      test('isNot true, incorrect cause', () => {
+        // only v16 or higher support Error.cause
+        if (Number(process.version.split('.')[0].slice(1)) >= 16) {
```

Comment:
can you use the helper? https://github.com/facebook/jest/blob/74776d8b581fabeb5c3993b6499643b8c6ffedf6/packages/test-utils/src/ConditionalTest.ts#L34